eclass/linux-mod-r1.eclass

```diff
@@ -1353,4 +1438,4 @@ _modules_update_depmod() {
 
 fi
 
-EXPORT_FUNCTIONS pkg_setup src_compile src_install pkg_postinst
+EXPORT_FUNCTIONS pkg_setup src_compile src_install pkg_postinst pkg_prerm
```

Comment:
I don't want to introduce new exports at this point without either doing linux-mod-r2 or doing it on EAPI bump, this eclass is meant to be stable at this point.